### DISCUSSION TITLE
What exactly does one row in .report.tsv represent?

### DISCUSSION THREAD

**Discussion by MatteoLacki** [vdemichev/DiaNN#333] [General]
Hi Vadim!

We have a question regarding the output of DiaNN.
We would basically like to check how MaxLFQ scores against other methods of protein quantification.
In order to do this, we need to get a good understanding of what is the precise definition of the each row in the output .report.tsv.

At least to me, it looks that each row corresponds to one quantified signal, i.e. each row corresponds to a cluster of points that share the same position in m/z, retention time, and inverse ion mobility in MS1 (timsTOF data is our principle point of interest), and that this cluster has been sequenced and can be assigned a set of possible protein origins (that are in Protein.Names).
Is that a proper understanding of what a row represents, or is there anything I missed?

If that was the case, one should only observe one occurrence of triplette (Run, Modified.Sequence, Precursor.Charge) with unique values. It seems to be the case on datasets we see: is that guaranteed?

Finally, let's assume I want to look at one peptide found in multiple Runs: which columns contain information about some retention-time aligned measurements? There are few to choose from: RT, RT.Start, RT.Stop, iRT, Predicted.RT, Predicted.iRT?
Or should one deduce the alignment by fitting some moving median to observed RTs and get these values oneself?

Very best wishes,

Matteo Lacki

**Comment by vdemichev** [comment]
Hi Matteo,

Each row is a precursor ID in a particular run, yes, guaranteed to be unique.
Predicted.iRT is the empirical retention time mapped to the iRT scale. You can view the alignment itself by plotting RT against iRT.

Best,
Vadim

**Reply by MatteoLacki** [reply]
Thanks!
